templates/index.html

```diff
@@ -0,0 +1,53 @@
+<!--index.html-->
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Cribbage Home Page</title>
+</head>
+
+<body>
+  <h1>Cribbage</h1>
+
+  <h2>Create user</h2>
+  <form>
+    <fieldset>
+      Username: <input type="text" name="username" id="createUN"><br>
+      Display name: <input type="text" id="createDN"><br>
+      <button disabled id="createUserButton">Create</button>
+    </fieldset>
+  </form>
+
+  <h2>Sign in</h2>
+  <form action="/user/" method="get">
+    <fieldset>
+      Username: <input type="text" name="username"><br>
+      <input type="submit" value="Submit">
+    </fieldset>
+  </form>
+
+  <script src="/assets/js/wasm_exec.js"></script>
+  <script>
+    const go = new Go();
+    // memoryBytes is an Uint8Array pointing to the webassembly linear memory.
+    let memoryBytes;
+    let mod, inst, bytes;
+    console.log("initializing gowasm");
+    WebAssembly.instantiateStreaming(
+      fetch("/assets/wasm/wa_output.wasm",
+        // change default to 'no-cache' while deving to force getting it
+        { cache: 'default' }),
+      go.importObject).then((result) => {
+        mod = result.module;
+        inst = result.instance;
+        memoryBytes = new Uint8Array(inst.exports.mem.buffer)
+        console.log("initialized gowasm");
+        run();
+      });
+    async function run() {
+      await go.run(inst);
+    }
+  </script>
```

Comment:
It seems like this is the same on every page. perhaps we should break it out into its own file?